wikidata.mediawiki

```diff
@@ -310,12 +311,12 @@ user_actions:
       operations:
         - # Appends a list item to the bottom of AIV
           type: append
-          page_name: Wikipedia:Administrator_intervention_against_vandalism
+          page_name: Administrators%27_noticeboard
```

Comment:
I don't know, I took the link that was in the URL, but I don't know what this program accepts...